site/content/integrate/apps/quick-start-js/_index.md

```diff
@@ -31,115 +29,114 @@ git clone https://github.com/mattermost/mattermost-plugin-apps.git
 Then build the plugin using:
 
 ```bash
+cd mattermost-plugin-apps
 make dist
 ```
 
-Then upload it via the System Console to your local Mattermost server.
+Then upload it to your local Mattermost server via the System Console.
 
 ## Building the app
 
-Start building your app by creating a directory for the code:
+Start building your app by creating a directory for the code, setup a new package and install  [`node-fetch`](https://www.npmjs.com/package/node-fetch) and [`express`](https://expressjs.com/), which will be used later:
 
 ```bash
 mkdir my-app
 cd my-app
+npm init
```

Comment:
Maybe suggest `npm init -y` to avoid the prompts with running this command

Reply:
I'm hesitant to use `-y` as the default suggested way. A dev can just spam enter anyway.